app.js

```diff
@@ -33,6 +34,31 @@ function runJava_2(clientId, main, files, timeLimitCompile, timeLimitExecution)
     });
 }
 
+function runJunit(messageId, tests, files, timeLimitCompile, timeLimitExecution) {
```

Comment:
document function args and return value properly
add the unit of time in the time* args